build/Nuget/Microsoft.NET.Nuget.proj

```diff
@@ -25,29 +21,31 @@
   <Target Name="Build"
           DependsOnTargets="LogOutput;
                             InvalidateNuGetCache;
-                            @(NuSpec->'%(LayoutTargetName)')"
-          Inputs="@(NuSpec);@(PackageAssets)"
-          Outputs="@(NuSpec->'%(NuPkgOutput)')">
+                            @(NuSpec->'%(LayoutTargetName)')">
```

Comment:
I think you should probably change `@(NuSpec->'%(LayoutTargetName)')` to `LayoutSdkPackage`, since you've gotten rid of the `LayoutTargetName` metadata on the `NuSpec` item.

EDIT: I see, you're specifying `BeforeTargets="Build"` on the `LayoutSdkPackage` target.  So if you want to leave it like that, you can remove this line.